rinex/src/navigation/ephemeris.rs

```diff
@@ -144,7 +144,14 @@ impl Ephemeris {
                 todo!("sv_clock_corr not supported for glonass @ the moment");
             },
             _ => {
-                let dt = (t - toe).to_seconds();
+                let mut dt = (t - toe).to_seconds();
+                // GPST ephemeris logic..
+                // TODO: does this apply to others like GST ?
+                if dt > 302400.0 {
+                    dt -= 604800.0;
+                } else if dt < -302400.0 {
+                    dt += 604800.0;
```

Comment:
Is this 7 days in seconds?

Reply:
Yes that's what it is, the SV express their time as a number of elapsed weeks, and seconds within the current week

Reply:
I'll see if I can use one of the #defines contained in Hifitime